### ISSUE TITLE
Error: Failed to calculate min for stats

### ISSUE THREAD

**Issue by ckhampus** [sitespeedio/sitespeed.io#2159]
Hi!
When running tests with `preURL` (or `preScript`) we get the following error:

```
Google Chrome 69.0.3497.81
Mozilla Firefox 62.0
[2018-10-02 08:45:21] INFO: Versions OS: linux 4.9.93-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.4.0 browsertime: 3.4.1 coach: 2.1.0
[2018-10-02 08:45:21] INFO: Starting chrome for analysing https://ckhampus.github.io/sitespeed-error-repro/ 3 time(s)
[2018-10-02 08:45:21] INFO: Testing url https://ckhampus.github.io/sitespeed-error-repro/ iteration 1
[2018-10-02 08:45:22] INFO: Accessing preURL https://ckhampus.github.io/sitespeed-error-repro/
[2018-10-02 08:45:32] INFO: BackEndTime: 6 DomInteractiveTime: 23 DomContentLoadedTime: 23 FirstPaint: 86 PageLoadTime: 74
[2018-10-02 08:45:32] INFO: VisualMetrics FirstVisualChange: 100 SpeedIndex: 111 PerceptualSpeedIndex: 104 LastVisualChange: 133
[2018-10-02 08:45:32] INFO: Testing url https://ckhampus.github.io/sitespeed-error-repro/ iteration 2
[2018-10-02 08:45:33] INFO: Accessing preURL https://ckhampus.github.io/sitespeed-error-repro/
[2018-10-02 08:45:43] INFO: BackEndTime: 10 DomInteractiveTime: 25 DomContentLoadedTime: 25 FirstPaint: 77 PageLoadTime: 55
[2018-10-02 08:45:43] INFO: VisualMetrics FirstVisualChange: 100 SpeedIndex: 111 PerceptualSpeedIndex: 104 LastVisualChange: 133
[2018-10-02 08:45:43] INFO: Testing url https://ckhampus.github.io/sitespeed-error-repro/ iteration 3
[2018-10-02 08:45:44] INFO: Accessing preURL https://ckhampus.github.io/sitespeed-error-repro/
[2018-10-02 08:45:54] INFO: BackEndTime: 6 DomInteractiveTime: 18 DomContentLoadedTime: 18 FirstPaint: 80 PageLoadTime: 59
[2018-10-02 08:45:54] INFO: VisualMetrics FirstVisualChange: 100 SpeedIndex: 111 PerceptualSpeedIndex: 104 LastVisualChange: 133
[2018-10-02 08:45:54] INFO: 2 requests, 380 bytes, backEndTime: 7ms (±1.09ms), firstPaint: 81ms (±2.16ms), firstVisualChange: 100ms (±0.00ms), DOMContentLoaded: 22ms (±1.70ms), Load: 63ms (±4.72ms), speedIndex: 111 (±0.00), visualComplete85: 133ms (±0.00ms), lastVisualChange: 133ms (±0.00ms), rumSpeedIndex: 81 (±2.16) (3 runs)
[2018-10-02 08:45:54] ERROR: Error: Failed to calculate min for stats: {
  "_config": {
    "store_data": true
  },
  "data": [
    null,
    112.946
  ],
  "length": 2,
  "sum": null,
  "sum_of_squares": null,
  "sum_of_logs": null,
  "sum_of_square_of_logs": null,
  "zeroes": 0,
  "min": null,
  "max": null,
  "_stddev": null,
  "_data_sorted": [
    null,
    112.946
  ]
}
    at percentiles.forEach.p (/usr/src/app/lib/support/statsHelpers.js:64:15)
    at Array.forEach (<anonymous>)
    at Object.summarizeStats (/usr/src/app/lib/support/statsHelpers.js:58:17)
    at reduce (/usr/src/app/lib/plugins/domains/aggregator.js:48:34)
    at /usr/src/app/node_modules/lodash.reduce/index.js:202:9
    at /usr/src/app/node_modules/lodash.reduce/index.js:1284:11
    at baseForOwn (/usr/src/app/node_modules/lodash.reduce/index.js:903:20)
    at /usr/src/app/node_modules/lodash.reduce/index.js:1253:14
    at baseReduce (/usr/src/app/node_modules/lodash.reduce/index.js:199:3)
    at reduce (/usr/src/app/node_modules/lodash.reduce/index.js:1842:10)
    at calc (/usr/src/app/lib/plugins/domains/aggregator.js:40:10)
    at Object.summarize (/usr/src/app/lib/plugins/domains/aggregator.js:118:16)
    at Object.processMessage (/usr/src/app/lib/plugins/domains/index.js:34:36)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
[2018-10-02 08:45:55] INFO: HTML stored in /sitespeed.io/sitespeed-result/ckhampus.github.io/2018-10-02-08-45-21
[2018-10-02 08:45:55] INFO: Finished analysing https://ckhampus.github.io/sitespeed-error-repro/
```

We started getting this issue when we added a Service Worker to our site, and we had difficulty to reproduce it. But I have since managed to reproduce the issue in the following https://github.com/ckhampus/sitespeed-error-repro on this url: https://ckhampus.github.io/sitespeed-error-repro/

I used the following command when running the tests:
```
docker run --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:7.4.0 https://ckhampus.github.io/sitespeed-error-repro/ -b chrome --preURL https://ckhampus.github.io/sitespeed-error-repro/
```

Thanks for the work you guys do!

**Comment by soulgalore** [comment]
Hi @ckhampus wow 1000*thanks for taking the time to create an example to recreate the bug, that is so valuable! 

I think there's a bug where we create the HAR from Chrome that sets one of the timings wrong. I'll work around that for now and just make that we even closer verify that all metrics are ok before we use them.

I'll merge the fix in master later today, then maybe you can try it out if you have time. 

Best
Peter

**Comment by soulgalore** [comment]
I've merged it now, in 10 minutes or so you can try it out by using 
sitespeedio/sitespeed.io-autobuild

`docker run --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io-autobuild https://ckhampus.github.io/sitespeed-error-repro/ -b chrome --preURL https://ckhampus.github.io/sitespeed-error-repro/`

**Comment by ckhampus** [comment]
It works now 👍 Thank you for the quick response on this issue 😄

**Comment by soulgalore** [comment]
Perfect, we will release it in a couple of days. The 10th of October it is 6 years ago we released 1.0 so we want to do the next release around that date :)

**Comment by ckhampus** [comment]
Awesome! Looking forward to it 🎉
